commenting_system/migrations/0002_test_data.py

```diff
@@ -3,28 +3,6 @@
 from Post.models import Post
 
 
-def generate_body_list():
-    body_list = [
-        'first comment test',
-        '~!@#$%^&*()_+/*-.:/<>;{}[]=`',
-        'another comment test. including special characters:!@#$%?',
-        'another comment test with a much more lines in body-' +
-        (20 * 'test \n') + 'test.',
-        'another comment test with a much larger line in body-' + (20 * 'test - ') + 'test']
-
-    return body_list
-
-
-def generate_user_list():
-    username_format = 'Test-user{}'
-    email_format = '{}@gmail.com'
-    password_format = '{}password'
-
-    return [User.objects.create_user(username_format.format(str(i)),
-                                     email_format.format(str(i)),
-                                     password_format.format(str(i))) for i in range(1, 6)]
-
-
 def generate_post(ind):
```

Comment:
This code is not called, please delete it

Reply:
Changed the file